apps/cowswap-frontend/src/modules/trade/pure/ProtocolFeeRow/index.tsx

```diff
@@ -38,10 +38,8 @@ export function ProtocolFeeRow({
       fiatAmount={protocolFeeUsd}
       tooltip={
         <Trans>
-          Protocol fee applied to this trade.
-          <br />
-          <br />
-          The fee is {protocolFeeBps} BPS ({protocolFeeAsPercent}%), applied only if the trade is executed.
+          The fee is {protocolFeeAsPercent}%, applied only if the trade is executed, in order to incentivize a strong
```

Comment:
One more suggestion from Anna

```suggestion
          The fee, applied only if the trade is executed, in order to incentivize a strong
```